spec/fixtures/pdf_fill/21P-527EZ/kitchen_sink.json

```diff
@@ -100,7 +100,7 @@
   ],
   "spouseDateOfBirth": "1960-01-01",
   "spouseSocialSecurityNumber": "333224444",
-  "spouseIsVeteran": true,
+  "spouseIsVeteran": false,
```

Comment:
we should revert this back to true, to stay in sync with the front-end fixtures.